README.md

```diff
@@ -17,41 +15,105 @@ Its design facilitates research and accelerates prototyping in the intersection
-- Visualization based on the [passive viewer][passive_viewer_mujoco] of Mujoco.
-
-### JaxSim as a simulator
-
-- Wide range of fixed-step explicit Runge-Kutta integrators.
-- Support for variable-step integrators implemented as embedded Runge-Kutta schemes.
-- Improved stability by optionally integrating the base orientation on the $\text{SO}(3)$ manifold.
-- Soft contacts model supporting full friction cone and sticking-slipping transition.
-- Collision detection between points rigidly attached to bodies and uneven ground surfaces.
-
-### JaxSim as a multibody dynamics library
+```
 
-- Provides rigid body dynamics algorithms (RBDAs) like RNEA, ABA, CRBA, and Jacobians.
-- Provides all the quantities included in the Euler-Poincarè formulation of the equations of motion.
-- Supports body-fixed, inertial-fixed, and mixed [velocity representations][notation].
-- Exposes all the necessary quantities to develop controllers in centroidal coordinates.
-- Supports running open-loop and full closed-loop control architectures on hardware accelerators.
+### Using JaxSim as a multibody dynamics library
+``` python
+import jax.numpy as jnp
+import jaxsim.api as js
+import icub_models
+import pathlib
+
+# Load the iCub model
+model_path = icub_models.get_model_file("iCubGazeboV2_5")
+joints = ('torso_pitch', 'torso_roll', 'torso_yaw', 'l_shoulder_pitch',
+          'l_shoulder_roll', 'l_shoulder_yaw', 'l_elbow', 'r_shoulder_pitch',
+          'r_shoulder_roll', 'r_shoulder_yaw', 'r_elbow', 'l_hip_pitch',
+          'l_hip_roll', 'l_hip_yaw', 'l_knee', 'l_ankle_pitch', 'l_ankle_roll',
+          'r_hip_pitch', 'r_hip_roll', 'r_hip_yaw', 'r_knee', 'r_ankle_pitch',
+          'r_ankle_roll')
+
+# Build and reduce the model
+model_description = pathlib.Path(model_path)
+full_model = js.model.JaxSimModel.build_from_model_description(
+    model_description=model_description, time_step=0.0001, is_urdf=True
+)
+model = js.model.reduce(model=full_model, considered_joints=joints)
+
+# Initialize model data
+data = js.data.JaxSimModelData.zero(model=model).reset_base_position(
+    base_position=jnp.array([0.0, 0.0, 1.0])
+)
+
+# Frame and dynamics computations
+frame_index = js.frame.name_to_idx(model=model, frame_name="l_foot")
+W_H_F = js.frame.transform(model=model, data=data, frame_index=frame_index)  # Frame transformation
+W_J_F = js.frame.jacobian(model=model, data=data, frame_index=frame_index)  # Frame Jacobian
+
+# Dynamics properties
+M = js.model.free_floating_mass_matrix(model=model, data=data)  # Mass matrix
+h = js.model.free_floating_bias_forces(model=model, data=data)  # Bias forces
+g = js.model.free_floating_gravity_forces(model=model, data=data)  # Gravity forces
+C = js.model.free_floating_coriolis_matrix(model=model, data=data)  # Coriolis matrix
+
+# Print dynamics results
+print(f"M: shape={M.shape}, h: shape={h.shape}, g: shape={g.shape}, C: shape={C.shape}")
```

Comment:
Collapsible section as above. Reformatted and used f-string syntax to print the shapes
```suggestion
<details>
<summary><b>Using JaxSim as a multibody dynamics library</b></summary>

``` python
import jax.numpy as jnp
import jaxsim.api as js
import icub_models
import pathlib

# Load the iCub model.
model_path = icub_models.get_model_file("iCubGazeboV2_5")
joints = ('torso_pitch', 'torso_roll', 'torso_yaw', 'l_shoulder_pitch',
          'l_shoulder_roll', 'l_shoulder_yaw', 'l_elbow', 'r_shoulder_pitch',
          'r_shoulder_roll', 'r_shoulder_yaw', 'r_elbow', 'l_hip_pitch',
          'l_hip_roll', 'l_hip_yaw', 'l_knee', 'l_ankle_pitch', 'l_ankle_roll',
          'r_hip_pitch', 'r_hip_roll', 'r_hip_yaw', 'r_knee', 'r_ankle_pitch',
          'r_ankle_roll')

# Build and reduce the model.
model_description = pathlib.Path(model_path)
full_model = js.model.JaxSimModel.build_from_model_description(
    model_description=model_description, time_step=0.0001, is_urdf=True
)
model = js.model.reduce(model=full_model, considered_joints=joints)

# Initialize model data.
data = js.data.JaxSimModelData.zero(model=model).reset_base_position(
    base_position=jnp.array([0.0, 0.0, 1.0])
)

# Frame and dynamics computations.
frame_index = js.frame.name_to_idx(model=model, frame_name="l_foot")
W_H_F = js.frame.transform(model=model, data=data, frame_index=frame_index)  # Frame transformation
W_J_F = js.frame.jacobian(model=model, data=data, frame_index=frame_index)  # Frame Jacobian

# Dynamics properties.
M = js.model.free_floating_mass_matrix(model=model, data=data)  # Mass matrix
h = js.model.free_floating_bias_forces(model=model, data=data)  # Bias forces
g = js.model.free_floating_gravity_forces(model=model, data=data)  # Gravity forces
C = js.model.free_floating_coriolis_matrix(model=model, data=data)  # Coriolis matrix

# Print dynamics results.
print(f"{M.shape=} \t {h.shape=} \t {g.shape=} \t {C.shape=}")
```

</details>
```

Reply:
same comments above about the collapse